blueprint/api.apib

```diff
@@ -336,13 +332,13 @@ To create a donation goal the user must be a site admin or the owner of the refe
 
 ### Filter donation goals by id [GET /donation-goals{?filter[id]}]
 
-+ Parameters
++ Request
 
-    + `filter[id]`: `1,2,3` (string, required) - Comma separated string of `ids` to filter by.
+    + Attributes
 
-        Technically, the parameter is `filter: { id: '1,2,3' }`, which ends up being serialized into `filter[id]`. However, syntax constraints do not allow us to define it in the documentation this way.
+        + `filter[id]`: `1,2,3` (string, required) - Comma separated string of `ids` to filter by.
```

Comment:
Actually I don't think this is how this should be written. I think it should still be a parameter.